providers/implementations/keymgmt/ml_kem_kmgmt.c

```diff
@@ -71,7 +71,7 @@ static int ml_kem_pairwise_test(const ML_KEM_KEY *key)
     int operation_result = 0;
 
     /* Unless we have both a public and private key, we can't do the test */
```

Comment:
Should the comment be updated to clarify that private key presence implies public key presence?

Reply:
Though the comment is true as written, it could perhaps be misconstrued, because in our ML-KEM implementation it is not possible to have just a private key, and this should be true fairly generally, as the serialised private key has the public key encoded in the middle of it. I am not particularly concerned about that comment in the PR, but it could be changed.